drafts/2024-09.md

```diff
@@ -42,6 +42,11 @@ and compute their values and derivatives. ndgrid is a library for handling finit
 In its first release, ndgrid supports grids of interval, triangle, and quadrilataral cells stored
 in serial or in parallel, with parallel mesh partitioning carried out using [coupe](https://crates.io/crates/coupe).
 
+### [ndarray v0.16.1](https://crates.io/crates/ndarray)
+ndarray implements an n-dimensional container for general elements and for numerics.
+This release mainly includes some refactors and bug fixes (see the [full release notes](https://github.com/rust-ndarray/ndarray/releases/tag/0.16.1)
+for more details).
+
```

Comment:
```suggestion
### [ndarray v0.16.0](https://crates.io/crates/ndarray)
Ndarray implements an n-dimensional container for general elements and for numerics.
Ndarray had its long outstanding v0.16.0 release recently which brought some larger changes and many bugfixes. Ndarray has since also followed up with some minor fixes in v0.16.1.
See the full release notes: [v0.16.0](https://github.com/rust-ndarray/ndarray/releases/tag/0.16.0), [v0.16.1](https://github.com/rust-ndarray/ndarray/releases/tag/0.16.0)
for more details).

```

Maybe it's ok to write it like this, mentioning the big release. Even if it's belated - it was in august, it's been years since the last 0.x release, so it merits mentioning?

Reply:
Oh yes sure! I have to admit I forgot there were two releases back to back 😅
The 0.16.0 is definitely worth mentioning.